.travis.yml

```diff
@@ -57,13 +57,29 @@ matrix:
       - MASTER=spark://127.0.0.1:7077
     install:
       - make build
+      - 
     script:
+      # start dependencies
       - ./scripts/get_apache_spark.sh "2.2.0" "2.7" || travis_terminate 1
       - ./scripts/start_apache_spark_cluster.sh "127.0.0.1" "7077" || travis_terminate 1
+      # python will be installed after https://github.com/src-d/gemini/issues/60
+      # and after merge https://github.com/src-d/gemini/pull/98 we can use local fe runner
```

Comment:
If this PR gets merged first - shall TODOs be added to #60 and #98 to take care of this?

Reply:
idk which one is going to be merged first. This PR contains lines that should be replaced later. That's why the comment is here.